packages/flutter/lib/src/material/expansion_panel.dart

```diff
@@ -253,6 +255,9 @@ class ExpansionPanelList extends StatefulWidget {
   /// {@macro flutter.material.ExpandIcon.color}
   final Color? expandIconColor;
 
+  /// size of the MaterialGap widget.
```

Comment:
seems like you haven't gone through the [Flutter Style guide](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo). Do give it a read, specially [this](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#documentation-dartdocs-javadocs-etc) section for documenting APIs.

generally, the format is like below. refer documentation of other members and methods for inspiration.

```dart
/// A short precise description.
///
/// More details disambiguating this API from similar ones, explaining edge cases, null or default behavior, etc.
///
/// See also section for related APIs.
```

Reply:
Sure I'll go through this. My bad.

Reply:
Done 👍✅ Please have a look.